crates/swcpack_node_interop/src/lib.rs

```diff
@@ -0,0 +1,161 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+use async_trait::async_trait;
+use helper::JsHook;
+use napi::{Env, JsFunction};
+use swc_common::{FileName, SourceMap};
+use swc_ecma_ast::EsVersion;
+use swc_ecma_parser::{parse_file_as_module, Syntax};
+use swcpack_core::{
+    asset::{AssetLoader, AssetLoaderContext, AssetProcessor},
+    esm::{EsModule, EsmPreprocessor, EsmPreprocessorContext, EsmProcessor},
+    file::FileLoader,
+    resource::{Res, ResourceIdGenerator},
+    Timer,
+};
+
+mod helper;
+
+pub struct JsAssetLoader {
```

Comment:
Maybe `EcmaAssetLoader`, because we use `ecma` everywhere for `js`? Same for `Js` everywhere to be align with our naming

Reply:
It's confusing as we have esm loader